README.md

```diff
@@ -373,6 +373,25 @@ In this example, we're updating the reservation with ID 1, changing the reserve_
 - GitHub: [@sagieramos](https://github.com/sagieramos)
 - Twitter: [@sagieramos](https://twitter.com/sagieramos)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/sagieramos)
+  
+👤 **Nilab Khwajazada**
```

Comment:
# Frontend re-review

- [ ] When I try to add an item with the included admin account in the seeds file, this happens:

![image](https://github.com/sagieramos/Book-an-appointment/assets/57421823/1bfead5a-bca6-4e5f-ae38-645045cf1c6e)

![image](https://github.com/sagieramos/Book-an-appointment/assets/57421823/199b8cfa-1fac-4482-bca0-0b6a5fbebc03)

- [ ] Trying to log out results in an error:

![image](https://github.com/sagieramos/Book-an-appointment/assets/57421823/50410449-7833-49c9-a79f-2336c2eb0f2a)

-  [ ] Every time Webpack compiles and runs the app, these linter errors happen:

![image](https://github.com/sagieramos/Book-an-appointment/assets/57421823/fb079652-40d9-42c2-bdd0-196850242135)

- [ ] The frontend repo still does not have a readme file.  Kindly, include one in order to improve your chances in the final evaluation.

Reply:
@Meltrust 
### For the frontend
- We have fixed the bug stopping a user from signing out
- Please delete the current `node_modules` and `package-lock.json`
- Run this command in your shell
```sh
yarn install
yarn start
```

### For the backend:
- The error signified that the database is not present in your environment
- Please, consider dropping your current database if you had already created it, then recreate it
```sh
  bundle install
  rails db:drop
  rails db:create
  rails db:migrate
  rails db:seed
```

### I have both the frontend and backend working in my environment 

![Screenshot from 2024-02-01 23-05-03](https://github.com/sagieramos/Book-an-appointment/assets/31627384/988964d1-9c09-4921-9b17-8d512ec94578)